src/convertToObject.js

```diff
@@ -12,7 +12,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const array = sourceString.split(';');
+  const resultConvert = {};
```

Comment:
`convertResult` will be better, but still bad
which convert? result of what? function result or temporary result?
`stylesObject` - will be better name